src/durableorchestrationcontext.ts

```diff
@@ -30,13 +30,18 @@ import {
 } from "./task";
 import moment = require("moment");
 import { ReplaySchema } from "./replaySchema";
-import { CallHttpOptions, Task, TimerTask } from "./types";
+import {
+    CallHttpOptions,
+    Task,
+    TimerTask,
+    DurableOrchestrationContext as DurableOrchestrationContextInterface,
+} from "./types";
 
 /**
  * Parameter data for orchestration bindings that can be used to schedule
  * function-based activities.
  */
-export class DurableOrchestrationContext {
+export class DurableOrchestrationContext implements DurableOrchestrationContextInterface {

```

Comment:
Seems we removed the DurableEntityContext file. Why?

Reply:
It was all just moved to the `entityTypes.ts` file, to get us one step closer to #458. The plan is to have all the types exported from the `types` directory, then convert the `.ts` files there into `.d.ts` files, and that will now become the only source `.d.ts` files.